common/icat/helpers.py

```diff
@@ -84,7 +84,7 @@ def logout_icat_client(client):
     :param client: ICAT client containing an authenticated user
     :type client: :class:`icat.client.Client`
     """
-
+    log.info("Logging out of the Python ICAT client")
```

Comment:
Why log here and not in backend.py like the other similar methods eg. login, refresh etc?

Reply:
Good spot of the inconsistency. I've only changed this instance because other instances of logging in `helpers.py` rather than `backend.py` are because the functions are used elsewhere (`get_entity_by_id()` is used when updating data for example). For this reason, only session related endpoints have logging in `backend.py`, so non-session endpoints are consistent in the fact their logging is in their respective helper functions.